osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-external-interface.xjb

```diff
@@ -0,0 +1,52 @@
+<jxb:bindings version="2.0" xmlns:jxb="http://java.sun.com/xml/ns/jaxb"
```

Comment:
There should be a copyright notice on top of this file. See the other binding files.

Reply:
Done